openy.profile

```diff
@@ -180,6 +180,7 @@ function openy_demo_content_configs_map($key = NULL) {
       'openy_demo_bsimple',
       'openy_demo_bamenities',
       'openy_demo_tfitness',
+      'y_lb_demo_content'
```

Comment:
@podarok please replace this migration group with the a new one `y_lb_init_demo_content`

Reply:
done